src/blog/2024-10-13/index.md

```diff
@@ -0,0 +1,117 @@
+"custom" crosshair have lots of settings typical for crosshair customization.
+
+The new convars for these settings are as follows:
+
+* `neo_cl_crosshair_style` - 0 by default, Set the crosshair style. 0 = default, 0 to 1 = textured, 2 = custom
+* `neo_cl_crosshair_color_[rgba]` - 255 by default, Set the [red/green/blue/alpha] value of the crosshair color
+* Only for custom-style crosshair:
+    * `neo_cl_crosshair_size_type` - 0 by default, Set the size unit used for the crosshair. `0 = neo_cl_crosshair_size, 1 = neo_cl_crosshair_size_screen`
+    * `neo_cl_crosshair_size` - 15 by default, Set the size of the crosshair
+    * `neo_cl_crosshair_size_screen` - 0 by default, Set the size of the crosshair by half-screen scale
+    * `neo_cl_crosshair_thickness` - 1 by default, Set the thickness of the crosshair
+    * `neo_cl_crosshair_gap` - 5 by default, Set the gap of the crosshair
+    * `neo_cl_crosshair_outline` - 0 by default, Set the crosshair outline
+    * `neo_cl_crosshair_center_dot` - 0 by default, Set the size of the center dot
+    * `neo_cl_crosshair_top_line` - 1 by default, Set if the top line will be shown
+    * `neo_cl_crosshair_circle_radius` - 0 by default, Set the circle radius of the crosshair
+    * `neo_cl_crosshair_circle_segments` - 30 by default, Set the segments of the circle
+
+### Ready up mechanic
+Ready up mechanic is now implemented, disabled by default. Use `neo_sv_readyup_lobby 1` server-side to enable it.
+Players can use the `!ready`/`!unready` commands to ready-up, as well as `!overridelimit` to allow for more players
+and `!playersnotready` to list players not ready.
+
+The new convars for these settings are as follows:
+
+* `neo_sv_readyup_lobby` - 0 by default, toggles the ready-up feature
+* `neo_sv_readyup_teamplayersthres` - 5 by default, the exact amount of players to ready-up and participate to start a game
+* `neo_sv_readyup_skipwarmup` - 1 by default, if ready-up feature is on and this is enabled, skip warmup state
+* `neo_sv_readyup_autointermission` - 0 by default, if disabled will not automatically enter intermission when the match ends
+
+
+### Player outlines in spectator mode
+Using the `+attack2` keybind (default `x`), can toggle between player outlines/indicators
+
+## Cloaking and visions
+
+* All visions (night vision, motion, and thermal) now have improved shaders which is now closer to OG:NT's visions
+* Cloaking material is now improved, tweaked values such as strength, speed, and distance effects more similar to OG:NT as well as when looked upon vision modes
+
+## General fixes/changes
+
+* Cleaner scope texturing and minor color adjustment
+* Workaround throwables (EX: grenades) being clipped to the ground, now can pick up grenades as normal
+* Grenade now throw closer to the center of the screen
+* Fixed general console errors, spams, and startup errors
+* Fixed wrong weapon set for the damage/killer information menu
```

Comment:
Fixed wrong weapon name used in the damage/killer

Not wrong but I think this is clearer